src/files-card-body.scss

```diff
@@ -202,8 +202,14 @@
             text-align: end;
         }
 
+        .col-perms, .item-perms {
+            inline-size: 17ch;
```

Comment:
dropped this in case it's not the right way do change this but with the otherwise set `inline-size: 12ch` it looks like this even with enough screen space:
![image](https://github.com/user-attachments/assets/33e65147-2bef-42d9-84eb-e9e094798608)

Reply:
This should be now obsolete no? As we added `width` to the `Th`?